plugins/workspace-minimap/README.md

```diff
@@ -30,9 +49,35 @@ const minimap = new Minimap(workspace);
 minimap.init();
 ```
 
+```css
+.blockly-minimap {
+  position: absolute;
+  box-shadow: none;
+  width: 200px;
+  height: 150px;
+  top: 0px;
+  left: 50vw;
+}
+```
+
+### Configuration
+The minimap takes a workspace as input and it inherits all of its properites (RTL, Theme, etc.).
+Additional styling of the minimap is possible with CSS. Use the `blockly-minimap` class for the minimap (box-shadow, etc.) and `blockly-focus-region` for the focus region (fill color, etc.).
+
+It is worth noting that when using a PositionedMinimap, the size of the minimap is determined by the window size, and the position by the primary workspace layout configuration. 
```

Comment:
I would move this up to the "positioned minimap" section, and remove the "It is worth noting" bit.